src/api/restaurants-api.ts

```diff
@@ -31,16 +31,15 @@ export const getMenuPerRestaurantQuery = (name: string) =>
 
 export const getMenuRestaurantQuery = (name: string) =>
```

Comment:
since this query is returning a single restaurant per name, we should rename it to something more accurate e.g. getRestaurantPerNameQuery